Dockerfile

```diff
@@ -17,7 +17,7 @@ COPY go.mod go.sum /go/node-feature-discovery/
 
 WORKDIR /go/node-feature-discovery
 
-RUN go mod download
+# RUN go mod download
```

Comment:
Why this?

Reply:
It sneaked into PR. I'll remove it right now.